shopfloor_account/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Shopfloor Account",
+    "summary": "Glue module between shopfloor and stock_account",
+    "version": "14.0.1.0.0",
+    "category": "Inventory, Accounting",
+    "website": "https://github.com/OCA/wms",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "maintainers": ["mmequignon"],
+    "license": "AGPL-3",
+    "installable": True,
+    "auto_install": False,
```

Comment:
```suggestion
    "auto_install": True,
```

Reply:
I would keep auto install False as this is not a glue module but an optional module